src/cascadia/TerminalApp/CascadiaSettings.cpp

```diff
@@ -182,16 +187,15 @@ void CascadiaSettings::_CreateDefaultSchemes()
 // - <none>
 void CascadiaSettings::_CreateDefaultProfiles()
 {
-    Profile cmdProfile{};
+    auto cmdProfile{ _CreateDefaultProfile(L"cmd") };

```

Comment:
Considering we _know_ what these are going to generate for GUIDs, to we want to paste those in the source here?

Reply:
we could. this DOES give us the freedom to change the algorithm later, though I reckon that would be bad for the same reason we need to make this fix

Reply:
I chose not to, because maybe we won't even need it in the future (defaults.json?)